.eslintrc

```diff
@@ -5,27 +5,20 @@
     "node": true
   },
   "plugins": [
-    "import",
     "testing-library",
     "@chanzuckerberg/stories"
   ],
   "extends": [
-    "eslint:recommended",
+    "@chanzuckerberg/eslint-config-edu-js",
     "@chanzuckerberg/eslint-config-edu-ts",
-    "plugin:jsx-a11y/recommended",
     "plugin:prettier/recommended",
-    "plugin:react/recommended",
```

Comment:
Not related to this line.

I originally had the shared prettier config in this PR, but it breaks the build 😑 

- It brings in `prettier-plugin-tailwindcss`
- which seems to cause Prettier to lint `tailwind.config.js` during the styles build
- which breaks because `tailwind.config.js` imports files that don't exist before the build completes
- in fact, at that point, the build can never complete, because it breaks because it's not complete :inception:

Removed from this PR, and we can look more into it later.